cdc/processor/processor.go

```diff
@@ -368,7 +368,7 @@ func (p *processor) handleTableOperation(ctx cdcContext.Context) error {
 				if opt.BoundaryTs < globalCheckpointTs {
 					log.Warn("the BoundaryTs of remove table operation is smaller than global checkpoint ts", zap.Uint64("globalCheckpointTs", globalCheckpointTs), zap.Any("operation", opt))
 				}
-				if !table.AsyncStop(opt.BoundaryTs) {
+				if !table.AsyncStop(p.changefeed.Status.ResolvedTs) {
```

Comment:
Why don't we modify the scheduler to ensure that the `global resolved ts` is used as the boundary to stop the table?